bindings/python/setup.py

```diff
@@ -172,8 +186,6 @@ def build_extension_with_b2(self):
 
         if self.parallel:
             args.append(f"-j{self.parallel}")
-        else:
-            args.append(f"-j{multiprocessing.cpu_count()}")
```

Comment:
is there a good reason not to do this?

Reply:
I just think it's standard / best practice to leave this sort of thing up to invocation.

Have you ever been watching a video that you care about more than your build, but the build makes the video stutter? This happens to me a lot :)

I pushed the `-j` invocation to `appveyor.yml`, with the intent to leave it there when i un-draft the PR.

Reply:
yes, but much more often I've built stuff with a single CPU instead of 30

Reply:
I think `ninja` is a good model

Reply:
Indeed. And I know newer golang defaults to using all cores. Maybe this preference is generational. I'll revert it.